app/serializers/appointment_serializer.rb

```diff
@@ -1,6 +1,37 @@
 class AppointmentSerializer < ActiveModel::Serializer
   attributes :id, :start_time, :status
 
+  CANCELED = :CANCELED
+  PAST = :PAST
+  PRESENT = :PRESENT
+  PENDING = :PENDING
+
+  def status
+    if object.is_canceled
+      return CANCELED
+    end
```

Comment:
Ruby has fancy way to such statements
```suggestion
    return CANCELED if object.is_canceled?
```